.github/workflows/tests.yml

```diff
@@ -33,3 +33,6 @@ jobs:
         run: python -m pip install ".[test]"
       - name: Test
         run: python -m pytest
+      - name: Test (pytest-run-parallel)
+        if: endsWith(matrix.python, 't') || endsWith(matrix.python, 't-dev')
+        run: python -m pytest --parallel-threads 4
```

Comment:
4 threads is informed by the number of cores on free github runners.
I've tested it locally on 32 threads on my 32-core box and it remains green.